andela/travel_tool_front
# #160715637 Check out of guesthouse

#### What does this PR do?
This provides an interface for a user to check out of a guesthouse

#### Description of Task to be completed?
As a requester, I should be prompted to check-out, on Travela, to confirm my departure from the guest house center
So that, travel team members at my destination and origin can be aware of my departure from the guest house centre

#### How should this be manually tested?
- Clone this repo - `git clone https://github.com/andela/travel_tool_front.git`
- Switch to the repo directory - `cd travel_tool_front`
- Check out this branch - `git checkout ft-checkout-guesthouse-160715637`
- Install dependencies - `yarn install`
- Start the server  - `make start`
- Set up the server by following this [guide](https://github.com/andela/travel_tool_back/pull/80)
- Navigate to `localhost:3000` in a browser
- Navigate to the booked rooms page using the link provided in the accommodation section on the left side navigation bar
- You should see your booked room record with a check-in button
- Click on the check-in button to check in to your guest house
- You should get a notification stating you just checked in to your guest house
- After a successful check-in, you should see a button that allows you to check out, click to checkout


#### Any background context you want to provide?
- After a successful checkout, the details of the trip then falls to past check-ins, this is so that a user can view details of places where they have checked in in time past.

#### What are the relevant pivotal tracker stories?
[160715637](https://www.pivotaltracker.com/story/show/160715637)


#### Screenshots (if appropriate)
<img width="1440" alt="screenshot 2018-10-07 at 10 48 23 pm" src="https://user-images.githubusercontent.com/22764463/46587457-8fb07480-ca84-11e8-8d61-6612a7a617b1.png">



#### Questions:
